src/components/data/data-utils.js

```diff
@@ -221,6 +222,13 @@ export const EntityDeleter = ({ onDismiss, onSuccess, namespace, name, selectedE
 
 const supportsFireCloudDataModel = entityType => _.includes(entityType, ['pair', 'participant', 'sample'])
 
+export const notifyDataImportProgress = (jobId) => {
```

Comment:
Nit: I believe our convention is to leave of the parens when it is possible to do so:
```suggestion
export const notifyDataImportProgress = jobId => {
```